idaes/models_extra/gas_solid_contactors/unit_models/moving_bed.py

```diff
@@ -143,6 +144,40 @@ class MBRData(UnitModelBlockData):
 and to "LAGRANGE-RADAU" for collocation transformation method,
 **"BACKWARD"** - Use a finite difference transformation method,
 **"FORWARD""** - use a finite difference transformation method,
+**"LAGRANGE-RADAU""** - use a collocation transformation method}""",
+        ),
+    )
+    CONFIG.declare(
+        "gas_transformation_scheme",
+        ConfigValue(
+            default=None,
```

Comment:
Seeing as this defaults to `"BACKWARD"` anyway, I would suggest that you just set the default value as such and remove `None` from the domain (that said, `None` is always a valid value for the domain, even if it is not explicitly declared).

Reply:
This doesn't default to `"BACKWARD"`, it defaults to whatever `transformation_scheme` is. Having `None` as the default is important as it lets us check whether these arguments are set.

Reply:
Ah - I see now. However, the doc string mentions Lagrange-Radau, which is not in the `In()` declaration.

Also, now that there are schemes for each side separately, is the declaration above still required (an overall transformation scheme)?

Reply:
Good catch. Lagrange-Radau is not supported if gas/solid are set individually. This is caught by the `transformation_method != "dae.finite_difference"` check on line 409, which is tested. I just updated the docstring.

`transformation_scheme` is still supported for backward compatibility, convenience, and to support collocation for the length domain (which may be useful for steady state).